src/kakarot/eth_rpc.cairo

```diff
@@ -0,0 +1,53 @@
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.uint256 import Uint256
+from starkware.starknet.common.syscalls import get_tx_info
+from kakarot.account import Account
+from kakarot.storages import Kakarot_native_token_address
+from kakarot.interfaces.interfaces import IAccount, IERC20
+from utils.utils import Helpers
+from utils.maths import unsigned_div_rem
```

Comment:
I know we have no linter for this but we (try to) follow the python convention, where external and internal libs are in two separate blocks